pkg/kubelet/kubelet.go

```diff
@@ -1943,3 +1947,222 @@ func (kl *Kubelet) GetRootInfo(req *cadvisorApi.ContainerInfoRequest) (*cadvisor
 func (kl *Kubelet) GetMachineInfo() (*cadvisorApi.MachineInfo, error) {
 	return kl.cadvisor.MachineInfo()
 }
+
+func (kl *Kubelet) SyncRocketPods(allPods []api.BoundPod, podSyncTypes map[types.UID]metrics.SyncPodType, start time.Time) error {
```

Comment:
@philips I meant what @yifan-gu suggested in his comment about the replacement. https://github.com/coreos/kubernetes/pull/4#discussion_r26350309